.vscode/tasks.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I think you didn't mean to commit this.

Reply:
Done. Thanks.
We shoudl consider putting a good one in.